src/ds5/ds5-options.cpp

```diff
@@ -373,13 +373,13 @@ namespace librealsense
         return *_range;
     }
 
-    external_sync_mode::external_sync_mode(hw_monitor& hwm)
-        : _hwm(hwm)
+    external_sync_mode::external_sync_mode(hw_monitor& hwm, sensor_base* ep, int ver)
```

Comment:
Can you encapsulate the [1,2,3] version into enum and document the modes for maintainers?
Similar to  `enum inter_cam_sync_mode`. 
The description in `external_sync_mode::get_description()` can be reused here

Reply:
Could you explain in more details what you like to do for introducing a new enum and where to put it? Note that enum inter_cam_sync_mode is in ds5-private.h. Do you like to include ds5-private.h in ds5-options.h? I added detailed comments in the constructor, is this enough?